storage/innobase/fil/fil0fil.cc

```diff
@@ -8332,8 +7923,9 @@ dberr_t Fil_shard::do_io(const IORequest &type, bool sync,
   }
 
   if (page_size.is_compressed()) {
-    req_type.mark_page_zip_compressed();
-    req_type.set_zip_page_physical_size(page_size.physical());
+    // TODO
+    //req_type.mark_page_zip_compressed();
+    //req_type.set_zip_page_physical_size(page_size.physical());
```

Comment:
@dutow, is it expected to stay like this for now?

Reply:
We don't need these, looks like I lost this change when I rebased to .31, I'll remove these lines.